src/main/java/net/aufdemrand/denizencore/tags/core/UtilTags.java

```diff
@@ -164,6 +164,17 @@ else if (attribute.startsWith("tau")) {
                     .getAttribute(attribute.fulfill(1)));
         }
 
+        // <--[tag]
+        // @attribute <util.e>
+        // @returns Element
+        // @description
+        // Returns e: 2.718281828459045
+        // -->
+        else if (attribute.startsWith("e")) {
```

Comment:
Should probably switch this to `attribute.matches` since it's literally 1 letter long.